boards/sodaq-sara-sff/include/board.h

```diff
@@ -146,6 +146,15 @@ extern "C" {
 #define SARA_R4XX_PWR_ON_OFF    (SARA_R4XX_PWR_ON_PORT.OUTCLR.reg = SARA_R4XX_PWR_ON_MASK)
 /** @} */
 
+/**
+ * @name    SARA (UBlox) STATUS
+ *
+ * This is the V_INT of the SARA.
+ * @{
+ */
+#define SARA_STATUS_PIN     GPIO_PIN(PA, 28)
```

Comment:
static test is failing again


```suggestion
#define SARA_STATUS_PIN     GPIO_PIN(PA, 28) /**< This is the V_INT of the board */
```

I *think* this would make static-test happy